ocaml/xapi/xapi_clustering.ml

```diff
@@ -545,6 +545,13 @@ module Watcher = struct
      is an update. *)
   let cluster_change_interval = Mtime.Span.(5 * min)
 
+  (* CA-396635: Sometimes it takes the underlying cluster stack (corosync) some time
+     to return a consistent view of the quorum. For example, it may be that the membership
+     information correctly reflects the new members after a membership change, while the
+     quorum field is still out of date. Add a delay here to make sure that the information
+     from corosync represents a consistent snapshot of the current cluster state. *)
+  let config_reform_timeout = Mtime.Span.(5 * s)
```

Comment:
This is not a timeout, the way I understand it - but a span for information to stabilize. So it's something that you would call a debounce period if this were a mechanical button.